core/network/src/protocol.rs

```diff
@@ -40,8 +42,12 @@ use client::light::fetcher::ChangesProof;
 use crate::{error, util::LruHashSet};
 
 const REQUEST_TIMEOUT_SEC: u64 = 40;
+/// Interval at which we perform time based maintenance
 const TICK_TIMEOUT: time::Duration = time::Duration::from_millis(1000);
```

Comment:
Does these two need to be round numbers? Currently status will fire at the same time as every fifth tick and every second propagate. Let's maybe use `1100` for tick and `2300` or `2900` for propagate? In general, if I'd say that `prime * 100` would be a good rule of thumb here unless we want something human-predictable like status printing.

Reply:
Ok!